<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configurações</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>631</width>
     <height>220</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QLineEdit" name="txtURL">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>82</y>
      <width>571</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>URL</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtUsername">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>140</y>
      <width>291</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>User Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtPassword">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>140</y>
      <width>261</width>
      <height>27</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cboLocalConexao">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>571</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>6</y>
      <width>221</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Local da Conexão</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>190</y>
      <width>48</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Padrão</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblIsDefault">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>190</y>
      <width>28</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Não</string>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frameBotoes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>631</width>
     <height>51</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="btnAplicar">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Aplicar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDefault">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Usar como Padrão</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
